src/components/ScholarshipForm.jsx

```diff
@@ -98,7 +98,6 @@ function ScholarshipForm() {
   return (
     <Container maxWidth="md">
```

Comment:
Now that this component is no longer a page component I suggest removing this as this is already in the page component.